src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothAvrcpControlImpl.cs

```diff
@@ -0,0 +1,491 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Tizen.Network.Bluetooth
+{
+    internal class BluetoothAvrcpControlImpl :IDisposable
+    {
+        private event EventHandler<PositionChangedEventArgs> _positionChanged;
+        private event EventHandler<PlayStateChangedEventArgs> _playStateChanged;
+        private event EventHandler<TrackInfoChangedEventArgs> _trackInfoChanged;
+        private event EventHandler<AvrcpControlConnChangedEventArgs> _connStateChanged;
+
+        private Interop.Bluetooth.PositionChangedCallback _positionChangedCallback;
+        private Interop.Bluetooth.PlayStatusChangedCallback _playStateChangedCallback;
+        private Interop.Bluetooth.TrackInfoChangedCallback _trackInfoChangedCallback;
+        private Interop.Bluetooth.AvrcpControlConnChangedCB _connStateChangedCallback;
+
+        private static BluetoothAvrcpControlImpl _instance = new BluetoothAvrcpControlImpl();
```

Comment:
Consider using lazy initialization

Reply:
_instance is created in constructor of BluetoothAvrcpControl. So, I think no need lazy initialization.